src/components/MuxedAccountResult.tsx

```diff
@@ -0,0 +1,47 @@
+import { Input } from "@stellar/design-system";
+import { useStore } from "@/store/useStore";
+
+import { PubKeyPicker } from "@/components/FormElements/PubKeyPicker";
+
+export const MuxedAccountResult = () => {
```

Comment:
The form doesn't work on the Create Muxed Account page. It might be better to make this component "dumb" and not connect it to the store. It can accept all three params needed and show what is passed. What do you think?

![image](https://github.com/stellar/laboratory/assets/7346473/a91ae08e-c57e-49e7-992d-34c897ed39d7)

Reply:
good point!